src/InvestJob.js

```diff
@@ -0,0 +1,119 @@
+import crypto from 'crypto';
+import localforage from 'localforage';
+
+import { getLogger } from './logger';
+
+const logger = getLogger(__filename.split('/').slice(-1)[0]);
+
+const KEYS_ARRAY = 'sortedHashArray';
+
+/**
+ * Create an object to hold properties associated with an Invest Job.
+ *
+ */
+export default class InvestJob {
+  /* If none exists, init an empty array for the sorted workspace hashes */
+  static async initDB() {
+    const keys = await localforage.getItem(KEYS_ARRAY);
+    if (!keys) {
+      localforage.setItem(KEYS_ARRAY, []);
+    }
+  }
+
+  /* Return an array of job metadata objects, ordered by most recently saved */
+  static async getJobStore() {
```

Comment:
Is the size of the job store limited to a certain number of jobs, or is `clearStore` ever automatically called? It seems like this could build up pretty fast

Reply:
Good point! I added a max of 30 jobs for now - always dropping the oldest one when a new one is saved after that. And also added a "clear" button in the settings dialog.